osu.Game/Overlays/Settings/Sections/Graphics/LayoutSettings.cs

```diff
@@ -230,6 +231,12 @@ private void updateScreenModeWarning()
                 return;
             }
 
+            if (RuntimeInfo.OS == RuntimeInfo.Platform.macOS)
+            {
+                windowModeDropdown.SetNoticeText(LayoutSettingsStrings.FullscreenMacOSNote, true);
```

Comment:
This is pretty contradictory with the message directly above. The game is going to tell the user to run fullscreen, then tell them not to.

I think the whole macOS flow should be separate, without the `NotFullscreenNote` above.